### PR TITLE
Zerihun.user profile

### PR DESCRIPTION
**Pull request by zerihunMoges**

### CODE REVIEW

File: starter-project-backend-g31/src/tests/integrationtest/user_profile.test.ts [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+              const { body, statusCode } = await supertest(app).post(
+                `/api/user-profiles`
+              ).send(userProfile);
+      
+              expect(statusCode).toBe(201);
+              expect(body._id).toBe(userId);
+            });
+      
+      });
+
```
Comment (RiyadHassen) [inline starter-project-backend-g31/src/tests/integrationtest/user_profile.test.ts:50]: add failure scenario for post condition

File: starter-project-backend-g31/src/tests/setupdb.ts [hunk 2]
```diff
@@ -21,7 +22,7 @@ export const connect = async () => {
 
 export const clear = async () => {
   const collections = await mongoose.connection.collections;
-
+  await UserProfile.deleteMany()
```
Comment (RiyadHassen) [inline starter-project-backend-g31/src/tests/setupdb.ts:25]: remove userProfile from UserProfiletest.

**Review by RiyadHassen — CHANGES_REQUESTED** [review]
Better to stick with project architecture(MVC), adding service is not important that much.

File: starter-project-backend-g31/src/tests/setupdb.ts (hunk 2 above)
Comment (zerihunttt655674) [reply starter-project-backend-g31/src/tests/setupdb.ts:25]: ok i used it because clear was not working

**Review by RiyadHassen — APPROVED** [review]
Looks Good To me

**Pull request merged**